common/php/trait-block-editor-compatible.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Experiment: a trait that abstracts the logic to overwrite anything that a specific module does.
+ * A prime use case would be not enqueueing EF classic editor assets.
+ *
+ */
+trait Block_Editor_Compatible {
```

Comment:
Traits are PHP 5.4+. We'll need to add a check in the main plugin file that skips loading the rest of the plugin for sites < PHP 5.4 (otherwise, we risk breaking those sites).

Example: https://github.com/ampproject/amp-wp/pull/891

Reply:
Added both premature return and Requires PHP header.